content/hi/docs/contribute/participate/roles-and-responsibilities.md

```diff
@@ -0,0 +1,141 @@
+---
+title: नियम और जिम्मेदारियाँ
+content_type: concept
+weight: 10
+---
+
+कुबेरनेट्स में कोई भी योगदान दे सकता है। जैसे-जैसे SIG Docs में आपका योगदान बढ़ता है, आप समुदाय में सदस्यता के विभिन्न स्तरों के लिए आवेदन कर सकते हैं। ये भूमिकाएं आपको समुदाय के भीतर अधिक जिम्मेदारी लेने की अनुमति देती हैं। प्रत्येक भूमिका के लिए अधिक समय और प्रतिबद्धता की आवश्यकता होती है। भूमिकाएँ हैं:
+
+- कोई भी: कुबेरनेट्स प्रलेखन में नियमित योगदानकर्ता
+- सदस्य: मुद्दों को असाइन और ट्राइएज कर सकते हैं और पुल अनुरोधों पर गैर-बाध्यकारी समीक्षा प्रदान कर सकते हैं
+- समीक्षक: प्रलेखन पुल अनुरोधों पर समीक्षा का नेतृत्व कर सकते हैं और परिवर्तन की गुणवत्ता के लिए पुष्टि कर सकते हैं
+- स्वीकृतकर्ता: दस्तावेज़ीकरण पर समीक्षा का नेतृत्व कर सकते हैं और परिवर्तनों को मर्ज कर सकते हैं
+
+## कोई भी
+GitHub खाते वाला कोई भी व्यक्ति कुबेरनेट्स में योगदान कर सकता है। SIG Docs सभी नए योगदानकर्ताओं का स्वागत करता है!
+
+- किसी भी [कुबेरनेट्स](https://github.com/kubernetes/) रिपॉजिटरी में एक समस्या खोलें, जिसमें शामिल हैं [`कुबेरनेट्स/वेबसाइट`](https://github.com/kubernetes/website)
+- पुल अनुरोध पर गैर-बाध्यकारी प्रतिक्रिया दें
+- स्थानीयकरण में योगदान करें
+- [Slack](https://slack.k8s.io/) या [SIG docs mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs) में सुधार का सुझाव दें
+
+[CLA पर हस्ताक्षर](https://github.com/kubernetes/community/blob/master/CLA.md)  करने के बाद, कोई भी यह कर सकता है:
+- मौजूदा सामग्री को बेहतर बनाने के लिए, नई सामग्री जोड़ने, या ब्लॉग पोस्ट या केस स्टडी लिखने के लिए पुल अनुरोध खोलें
+- डायग्राम, ग्राफिक्स एसेट और एम्बेड करने योग्य स्क्रीनकास्ट और वीडियो बनाएं
+
+अधिक जानकारी के लिए [नई सामग्री का योगदान](/docs/contribute/new-content/) देखें|
+
+## सदस्यों
+सदस्य वह है जिसने `कुबेरनेट्स/वेबसाइट` पर एकाधिक पुल अनुरोध सबमिट किए हैं| 
```

Comment:
You've missed a line of translation here. Please refer to the English documentation.